.idea/.name

```diff
@@ -1 +1 @@
-WoWAnalyzer
\ No newline at end of file
+AlwaysBeCasting.tsx
```

Comment:
... mistaken addition?

Reply:
Deleted it now, not sure what it is/why it is there